### PR TITLE
CT: Add IPEDS to Caution Flag Mappings #18306

### PR DESCRIPTION
**Pull request by zurbergram**
## Original issue(s)
department-of-veterans-affairs/va.gov-team#18306

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: app/models/institution_builder.rb [hunk 1]
```diff
@@ -483,6 +506,39 @@ def self.add_settlement(version_id)
...
+              #{flag_date_sql} as flag_date,
+              #{conn.quote(timestamp)} as created_at,
+              #{conn.quote(timestamp)} as updated_at
+	        FROM institutions JOIN (
+            SELECT "cross" as ipeds, settlement_title, settlement_description, settlement_date, settlement_link
+            FROM institutions JOIN va_caution_flags ON institutions.facility_code = va_caution_flags.facility_code
+            WHERE settlement_title IS NOT NULL
+              AND settlement_description IS NOT NULL
+              AND "cross" IS NOT NULL
+              AND "cross" = '180814'
```
Comment (shawkey-daniel) [inline app/models/institution_builder.rb:534]: Only for 180814?

**Review by shawkey-daniel — CHANGES_REQUESTED** [review]

File: app/models/institution_builder.rb (hunk 1 above)
Comment (zurbergram) [reply app/models/institution_builder.rb]: that fixed my unit test

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by delli-gatti-michael — APPROVED** [review]

**Review by thilton-oddball — APPROVED** [review]
LGTM

**Pull request merged**